library/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ set(F3D_VTK_MODULES
   VTK::CommonColor
   VTK::IOImage
   VTK::InteractionWidgets
+  VTK::RenderingGridAxes
```

Comment:
here you need to do something similar to raytracing but based on the vtk version, eg:

```
if(VTK_VERSION VERSION_GREATER_EQUAL 9.4.20250513)
  set(F3D_VTK_MODULES ${F3D_VTK_MODULES} VTK::RenderingRayTracing)
endif()
```